etc/tags.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -ex
+
+#
+# This little script tries guessing which executable
+# in the system is exuberant ctags. It first checks whether
+# the environment variable $CTAGS is set, and if so, executes
+# the command given by it, otherwise it first tries locating
+# `exctags`, and then `ctags`.
+#
+# Of course there is a lot of room for sophistication here,
+# but this works for me at the moment.
+#
+
+if [[ x$CTAGS != "x" ]]
+then
+    $CTAGS $@
+    exit
+fi
```

Comment:
Careful: exuberant ctags reads options from the `CTAGS` environment variable, so you may want to use something else here.